sandpack-react/src/presets/Sandpack.stories.tsx

```diff
@@ -286,3 +290,39 @@ export const ShowConsoleButton: React.FC = () => (
     />
   </div>
 );
+
+export const CustomLanguages: React.FC = () => (
+  <Sandpack
+    additionalLanguages={[
```

Comment:
I'd add the `additionalLanguages` inside of `options. codeEditor`, which would make clearer where this key belongs